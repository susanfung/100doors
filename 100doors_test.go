package main

import (
	"testing"
	approvals "github.com/approvals/go-approval-tests"
)

func TestInitialState(t *testing.T) {
	approvals.VerifyArray(t, Answer(0))
}

func TestFirstPass(t *testing.T) {
	approvals.VerifyArray(t, Answer(1))
}

func TestSecondPass(t *testing.T) {
	approvals.VerifyArray(t, Answer(2))
}