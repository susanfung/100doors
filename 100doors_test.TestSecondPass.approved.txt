[0]=opened
[1]=closed
[2]=opened
[3]=closed
[4]=opened
[5]=closed
[6]=opened
[7]=closed
[8]=opened
[9]=closed
[10]=opened
[11]=closed
[12]=opened
[13]=closed
[14]=opened
[15]=closed
[16]=opened
[17]=closed
[18]=opened
[19]=closed
[20]=opened
[21]=closed
[22]=opened
[23]=closed
[24]=opened
[25]=closed
[26]=opened
[27]=closed
[28]=opened
[29]=closed
[30]=opened
[31]=closed
[32]=opened
[33]=closed
[34]=opened
[35]=closed
[36]=opened
[37]=closed
[38]=opened
[39]=closed
[40]=opened
[41]=closed
[42]=opened
[43]=closed
[44]=opened
[45]=closed
[46]=opened
[47]=closed
[48]=opened
[49]=closed
[50]=opened
[51]=closed
[52]=opened
[53]=closed
[54]=opened
[55]=closed
[56]=opened
[57]=closed
[58]=opened
[59]=closed
[60]=opened
[61]=closed
[62]=opened
[63]=closed
[64]=opened
[65]=closed
[66]=opened
[67]=closed
[68]=opened
[69]=closed
[70]=opened
[71]=closed
[72]=opened
[73]=closed
[74]=opened
[75]=closed
[76]=opened
[77]=closed
[78]=opened
[79]=closed
[80]=opened
[81]=closed
[82]=opened
[83]=closed
[84]=opened
[85]=closed
[86]=opened
[87]=closed
[88]=opened
[89]=closed
[90]=opened
[91]=closed
[92]=opened
[93]=closed
[94]=opened
[95]=closed
[96]=opened
[97]=closed
[98]=opened
[99]=closed